compiler/rustc_session/src/options.rs

```diff
@@ -1033,6 +1033,8 @@ options! {
         "emit bitcode in rlibs (default: yes)"),
     extra_filename: String = (String::new(), parse_string, [UNTRACKED],
         "extra data to put in each output filename"),
+    force_allocator_shim: Option<bool> = (None, parse_opt_bool, [TRACKED],
+        "force generation of an allocator shim"),
```

Comment:
Please use `-Z` instead of `-C` to avoid immediately stabilizing this. This option is only useful when depending on an implementation detail (that rlibs are currently archives) anyway.

Reply:
Do you have a code pointer for how to make this a `-Z` option or gate it behind `-Z unstable-features`? I wasn't able to find that on my own (it's one of the TODOs in the original comment)